### DISCUSSION TITLE
French traduction

### DISCUSSION THREAD

**Discussion by Lurikbolding** [aclap-dev/video-downloadhelper#1293] [Language & Translation]
Hello

Here is my proposition for the french traduction.
[langage-french.txt](https://github.com/user-attachments/files/17099599/langage-french.txt)

**Comment by MiLoux** [comment]
idem ...
[video dwld helper fr.txt](https://github.com/user-attachments/files/17103880/video.dwld.helper.fr.txt)

**Comment by 1000vario** [comment]
Here is my proposition for the french traduction up to date.
[langage-french.txt](https://github.com/user-attachments/files/17289357/langage-french.txt)
